dev/TabView/TabView.idl

```diff
@@ -46,6 +46,7 @@ runtimeclass TabViewTabDragCompletedEventArgs
 
 [WUXC_VERSION_MUXONLY]
 [webhosthidden]
+[contentproperty("TabItems")]
```

Comment:
it surprises me that this doesn't come with any generated files changes

Reply:
I believe the way this works is that this attribute gets tacked onto the winmd metadata and the parser can then get this information from the metadata. @danzil might be able to elaborate more.